### PR TITLE
delete pip section

### PR DESCRIPTION
**Pull request by sapirshuker**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes:  https://jira-hq.paloaltonetworks.local/browse/CIAC-3838
           https://jira-hq.paloaltonetworks.local/browse/CIAC-7635
## Description
All the pip and pipfile.lock files have been deleted from the content repository, there is no need for the *Setup Pipenv* section anymore.

### CODE REVIEW

File: docs/integrations/unit-testing.md [hunk 1]
```diff
@@ -9,38 +9,28 @@ Unit testing should be used to test small units of code in an isolated and deter
...
-We recommend using VSCode with the Cortex XSOAR Extension This is optional and you can also run/debug unit tests with other IDEs (such as Pycharm).
+We recommend using VSCode with the Cortex XSOAR Extension This is optional and you can also run/debug unit tests with other IDEs (such as Pycharm). 
 
 Setup:
 
 * **Install the Cortex XSOAR Extension**: Install with-in VSCode by navigating to `Extension`. Or download and install from [here](https://marketplace.visualstudio.com/items?itemName=CortexXSOARext.xsoar)
 * **Open VSCode**: Open VSCode where the root folder is the folder you wish to develop within.
-* **Choose Interpreter**: Choose the Pipenv interpreter (with all dependencies we setup in the previous step). See: <https://code.visualstudio.com/docs/python/environments>
 * **Enable PyTest**: We run our unit tests with `pytest`. See the following on how to enable PyTest: <https://code.visualstudio.com/docs/python/testing>
+* **Setup Integration And Script Environment**: form more details, check out [setup integrations and scripts environment](https://xsoar.pan.dev/docs/concepts/vscode-extension#setup-integrations-and-scripts-environment).
```
Comment (ShirleyDenkberg) [inline docs/integrations/unit-testing.md:25]:
```suggestion
* **Setup Integration And Script Environment**: for more details, check out [setup integrations and scripts environment](https://xsoar.pan.dev/docs/concepts/vscode-extension#setup-integrations-and-scripts-environment).
```

File: docs/integrations/unit-testing.md [hunk 2]
```diff
@@ -9,38 +9,27 @@ Unit testing should be used to test small units of code in an isolated and deter
...
-* **Install additional runtime dependencies**: using: `pipenv install <dependency>`. For example: `pipenv install ldap3`
-* **Sync Pipenv**: (including dev dependencies) by running: `pipenv sync --dev`
-* **Enable Virtual Env**: To enable the Pipenv virtual env in the shell run: `pipenv shell`. To exit the virtual env simply run: `exit`.
-
-You should now have a managed virtual environment to run unit tests locally.
+### General Notes
+In order to verify that the content indeed runs with all the required dependencies, we recommend writing, running, and debugging the unit tests locally with the corresponding image. 
+The VSCode Cortex XSOAR Extension allows you to do this in the easiest way possible. Additionally, you can run and debug the unit tests locally with the corresponding image in other IDEs.
 
 ### Setup Vscode
```
Comment (ShirleyDenkberg) [inline docs/integrations/unit-testing.md:16]:
```suggestion
### Setup Dev Environment 
```

File: docs/integrations/unit-testing.md [hunk 3]
```diff
@@ -9,38 +9,27 @@ Unit testing should be used to test small units of code in an isolated and deter
...
 
-We recommend using VSCode with the Cortex XSOAR Extension This is optional and you can also run/debug unit tests with other IDEs (such as Pycharm).
+We recommend using VSCode with the Cortex XSOAR Extension This is optional and you can also run/debug unit tests with other IDEs (such as Pycharm).
```
Comment (ShirleyDenkberg) [inline docs/integrations/unit-testing.md:18]:
```suggestion
We recommend using VSCode with the Cortex XSOAR Extension This is optional and you can also run/debug unit tests with other IDEs (such as Pycharm).  For more information, see (Environment setup](https://xsoar.pan.dev/docs/concepts/vscode-extension#environment-setup).
```

File: docs/integrations/unit-testing.md [hunk 4]
```diff
@@ -9,38 +9,27 @@ Unit testing should be used to test small units of code in an isolated and deter
...
+### General Notes
+In order to verify that the content indeed runs with all the required dependencies, we recommend writing, running, and debugging the unit tests locally with the corresponding image. 
+The VSCode Cortex XSOAR Extension allows you to do this in the easiest way possible. Additionally, you can run and debug the unit tests locally with the corresponding image in other IDEs.
 
 ### Setup Vscode
 
-We recommend using VSCode with the Cortex XSOAR Extension This is optional and you can also run/debug unit tests with other IDEs (such as Pycharm).
+We recommend using VSCode with the Cortex XSOAR Extension This is optional and you can also run/debug unit tests with other IDEs (such as Pycharm). 
 
 Setup:
```
Comment (ShirleyDenkberg) [inline docs/integrations/unit-testing.md:20]:
```suggestion

```

File: docs/integrations/unit-testing.md (hunk 2 above)
Comment (sapirshuker) [reply docs/integrations/unit-testing.md:16]: We need to distinguish between the setup of the the user can setup the dev environment with Vscode or with Pycharm

**Review by ilaner — APPROVED** [review]

**Pull request merged**